WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/Add Product Variation/AddAttributeViewModelTests.swift

```diff
@@ -65,6 +65,37 @@ final class AddAttributeViewModelTests: XCTestCase {
         XCTAssertTrue(result)
         XCTAssertEqual(storesManager.numberOfResponsesConsumed, 1)
     }
+
+    func test_handle_valid_new_attribute_name() throws {
```

Comment:
nit: this test case probably doesn't throw?

```suggestion
    func test_handle_valid_new_attribute_name() {
```